packages/release-mastermind/src/conditions/util/isStale.ts

```diff
@@ -0,0 +1,22 @@
+import { UtilProps, UtilThis } from "..";
+
+const TYPE = "isStale";
+
+export interface ConditionIsStale {
+  type: typeof TYPE;
```

Comment:
Codacy has a fix for the issue: [A readonly modifier is required.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
  readonly type: typeof TYPE;
```